### DISCUSSION TITLE
Check failing due to IAM Roles created by AWS Control Tower and AFT with AdministratorAccess  policy

### DISCUSSION THREAD

**Discussion by dmkim22-lguplus** [prowler-cloud/prowler#3809] [Q&A] [answered]
Hello,

IAM Roles created by AWS Control Tower and AFT (Account Factory for Terraform) have AdministratorAccess policy attached, and it seems "Ensure IAM AWS-Managed policies that allow full \"*:*\" administrative privileges are not attached" is failing due to this.

Should the following IAM Roles in an account be excluded from this check?

- aws-controltower-AdministratorExecutionRole
- AWSAFTExecution
- AWSAFTService
- AWSControlTowerExecution
- stacksets-exec-*

Thank you in advance.

**Comment by jfagoagas** [comment, marked as answer]
@dmkim22-lguplus I'm moving this to an issue https://github.com/prowler-cloud/prowler/issues/3810

Thanks for using Prowler 🚀
